src/CanvasContainer.tsx

```diff
@@ -198,22 +427,11 @@ export const CanvasContainer: React.FC = ({ children }) => {
    */
   useEffect(() => {
     const onCanvasWheel = (e: WheelEvent) => {
-      const isWithMetaKey = isWithPlatformMetaKey(e);
+      const isZoomEnabled = !embed?.isEmbedded || embed.zoom;
+      const isPanEnabled = !embed?.isEmbedded || embed.pan;
```

Comment:
isn't it easier to think of these 2 conditions from the negative side?
```js
const isZoomDisabled = embed?.isEmbedded && !embed.zoom;
```
just reads better than a positive check.

Reply:
I just prefer to use a single flavor across the app - especially once you start creating utils/selectors for this kind of stuff and also to avoid double negation with stuff like `!isZoomDisabled`. "not not" seems slightly harder to read at times (it isn't particularly hard to read here though). 

I don't mind changing it - let me know what do you think

Reply:
I don't have a strong opinion over this. I'm in the minority who prefer negative variable names for toggle states.